cmd/kill/kill.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"os"
+	"strconv"
+	"fmt"
+	"flag"
+
+	killer "github.com/tiago4orion/enzo/cmd/kill/killer"
+)
+
+// save own name for easier reference
+var me = os.Args[0]
+
+func usage() {
+	fmt.Println("Usage:")
+	fmt.Println("kill [-safe] pids")
+	flag.PrintDefaults()
+	os.Exit(22); // EINVAL
+}
+
+func sliceatoi(ss []string) ([]int, error) {
+	is := make([]int, 0, len(ss))
+
+	for _, s := range ss {
+		i, err := strconv.Atoi(s)
+		if err != nil {
+			return []int{}, err
+		}
+		is = append(is, i)
+	}
+	return is, nil
+}
+
+func parseargs() ([]int, bool) {
+	var safe bool
+
+	flag.BoolVar(&safe, "safe", false, "doesn't use SIGKILL when SIGTERM fail (unix systems)")
+	flag.Parse()
+
+	pids, err := sliceatoi(flag.Args())
+	if err != nil || len(pids) == 0 {
+		usage()
```

Comment:
Well, as it is usage() will always Exit(). But we can improve that letting usage() to be called in case of `kill -h`. For that parseargs() must return Error.New() when len(pids) == 0.